### DISCUSSION TITLE
Shelly Wave restore state after a power failure not working

### DISCUSSION THREAD

**Discussion by pm-melo** [zwave-js/zwave-js#7499] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

**What is happening?**
Hi everyone! I'm using a Shelly Wave 1PM with a light connected to it. The device appears to work correctly apart from parameter 17 of the device. According to the manufacturer specification ([link](https://us.shelly.com/blogs/documentation/wave-1pm-us)) - this parameter should allow for a state restore after power is cut to the device. I'm not seeing this work. To me, after a power loss, the device always becomes off.

**What did you expect to happen instead?**
The device should restore the switch state after a power loss.

**Steps to reproduce the behavior:**
  1. Go to 'Control Pannel'
  2. Click on the device to expand
  3. Scroll down to 'Configuration V4', set the parameter '[6-112-0-17] O1: State After Power Failure' to 0 (should make the device restore the state according to the manufacturer)
  4. Turn the device switch on. Disconnect the device from power. Wait a few seconds. Connect power again and validate that the devices stays off.

### Software versions

zwave-js-ui: 9.28.0.e336517
zwave-js: 14.3.7
home id: 3817940233
home hex: 0xe3912509

### Z-Wave Controller (Stick/Dongle/...)

I'm using a Zooz 800 USB stick

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js.log](https://github.com/user-attachments/files/18135793/zwave_js.log)

**Comment by AlCalzone** [comment, marked as answer]
I'm currently testing Shelly devices anyways. Will make sure to keep this in mind and report back to Shelly if I can reproduce.
